registration_login/templates/registration_login/register.html

```diff
@@ -0,0 +1,78 @@
+    <div class="container-fluid">
+      <div class="row  justify-content-center align-items-center d-flex-row text-center h-100">
+        <div class="col-12 col-md-4 col-lg-3   h-50 ">
+          <div class="card shadow">
+            <div class="card-body mx-auto">
+              <h4 class="card-title mt-3 text-center">Stwórz konto</h4>
+              <p class="text-center">Załóż swoje darmowe konto</p>
+              <form method="POST" action="">
+                {% csrf_token %}
+                <div class="form-group input-group">
+                  <div class="input-group-prepend">
+                    <span class="input-group-text"> <i class="fa fa-user"></i> </span>
+                  </div>
+                  {{form.username}}
+                </div>
+                <div class="form-group input-group">
+                  <div class="input-group-prepend">
+                    <span class="input-group-text"> <i class="fa fa-envelope"></i> </span>
+                  </div>
+                  {{form.email}}
+                </div>
+                <div class="form-group input-group">
+                  <div class="input-group-prepend">
+                    <span class="input-group-text"> <i class="fa fa-lock"></i> </span>
+                  </div>
+                  {{form.password1}}
+                </div>
+                <div class="form-group input-group">
+                  <div class="input-group-prepend">
+                    <span class="input-group-text"> <i class="fa fa-lock"></i> </span>
+                  </div>
+                  {{form.password2}}
+                </div>
+                <div class="form-group">
+                  <button type="submit" class="btn btn-primary btn-block"> Stwórz konto </button>
+                </div>
+                {{form.errors}}
+                <p class="text-center">Masz już konto?
+                  <a href="{% url 'login' %}">Zaloguj się</a>
+                </p>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+ </section>
+
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+<script>
+						
+        
+    var form_fields = document.getElementsByTagName('input')
```

Comment:
teraz raczej używa się `let`, abstrahując od tego, że najpewniej należałoby całkiem inaczej tego jsa napisać, aczkolwiek to nie jest główny topic kursu :)